### PR TITLE
fix(components): [table] tfoot is not same table with tbody when use tablelayout

### LINKED ISSUE #9957: [Component] [table] tfoot is not same table with tbody when use table-layout=auto
Bug Type: **`Component`**

## Environment

- Vue Version: `3.2.29`
- Element Plus Version: `2.2.17`
- Browser / OS: `Chrome 105.0.0.0 / Window 11`
- Build Tool: `Vite`

### Related Component

- `el-table`

### Reproduction Link

[Docs](https://element-plus.org/en-US/component/table.html#summary-row)

### Steps to reproduce

<el-table
	:data="items"
	:default-sort="{ prop: 'date', order: 'descending' }"
	:summary-method="getSummaries"
	show-summary
	table-layout="auto"
>

### What is Expected?

The column in the tfoot aligns with the column above

### What is actually happening?

The column in the tfoot doesn't aligns with the column above

### Additional comments

Because tfoot is in table in "el-table__footer-wrapper", not same table in "el-table__body-wrapper".
when I use table layout auto + show-summary, columns at tfoot are not aligned

### PR DESCRIPTION
**Pull request by tctci**
fix #9957

before
641a88d19e7f94becae7788013cce87e935384d6dc2824eb3ca327f972f5cdf8QzpcVXNlcnNcQWRtaW5pc3RyYXRvclxBcHBEYXRhXFJvYW1pbmdcRGluZ1RhbGtcMjIxNTQxNTI1NV92MlxJbWFnZUZpbGVzXDE2NjQ1MTc1NTA1OTZfMUNFNzMxNzItNTc2RS00MWYwLTk4QzAtMjM0RkMwMjREOTc3LnBuZw==
after
d22d9f2081bf6c3d8dd81f2c3d6bb882f9f55dabe91675e6bc269c1e94bec261QzpcVXNlcnNcQWRtaW5pc3RyYXRvclxBcHBEYXRhXFJvYW1pbmdcRGluZ1RhbGtcMjIxNTQxNTI1NV92MlxJbWFnZUZpbGVzXDE2NjQ1MTU2ODQ3MTBfNTBGQjc5NzktNjRERS00ZTFlLThCRkQtRjg5NDVCODhGMDQzLnBuZw==
Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: packages/components/table/src/table.vue [hunk 1]
```diff
@@ -96,6 +96,21 @@
...
+              :border="border"
+              :default-sort="defaultSort"
+              :store="store"
+              :class="ns.e('footer-wrapper')"
+              :style="{
+                ...tableBodyStyles,
+              }"
+              :sum-text="computedSumText"
+              :summary-method="summaryMethod"
+            />
```
Comment (tolking) [inline packages/components/table/src/table.vue:113]: When using the `height="200" + table-layout="fixed"`, the `table-footer` of the table is not fixed to the bottom as before.  [Demo](https://element-plus.run/#eyJBcHAudnVlIjoiPHRlbXBsYXRlPlxuICA8ZWwtdGFibGVcbiAgICA6ZGF0YT1cInRhYmxlRGF0YVwiXG4gICAgYm9yZGVyXG4gICAgdGFibGUtbGF5b3V0PVwiZml4ZWRcIlxuICAgIGhlaWdodD1cIjIwMFwiXG4gICAgOnN1bW1hcnktbWV0aG9kPVwiZ2V0U3VtbWFyaWVzXCJcbiAgICBzaG93LXN1bW1hcnlcbiAgICBzdHlsZT1cIndpZHRoOiAxMDAlOyBtYXJnaW4tdG9wOiAyMHB4XCJcbiAgPlxuICAgIDxlbC10YWJsZS1jb2x1bW4gcHJvcD1cImlkXCIgbGFiZWw9XCJJRFwiIHdpZHRoPVwiMTgwXCIgLz5cbiAgICA8ZWwtdGFibGUtY29sdW1uIHByb3A9XCJuYW1lXCIgbGFiZWw9XCJOYW1lXCIgLz5cbiAgICA8ZWwtdGFibGUtY29sdW1uIHByb3A9XCJhbW91bnQxXCIgbGFiZWw9XCJDb3N0IDEgKCQpXCIgLz5cbiAgICA8ZWwtdGFibGUtY29sdW1uIHByb3A9XCJhbW91bnQyXCIgbGFiZWw9XCJDb3N0IDIgKCQpXCIgLz5cbiAgICA8ZWwtdGFibGUtY29sdW1uIHByb3A9XCJhbW91bnQzXCIgbGFiZWw9XCJDb3N0IDMgKCQpXCIgLz5cbiAgPC9lbC10YWJsZT5cbjwvdGVtcGxhdGU+XG5cbjxzY3JpcHQgbGFuZz1cInRzXCIgc2V0dXA+XG5pbXBvcnQgdHlwZSB7IFRhYmxlQ29sdW1uQ3R4IH0gZnJvbSAnZWxlbWVudC1wbHVzL2VzL2NvbXBvbmVudHMvdGFibGUvc3JjL3RhYmxlLWNvbHVtbi9kZWZhdWx0cydcblxuaW50ZXJmYWNlIFByb2R1Y3Qge1xuICBpZDogc3RyaW5nXG4gIG5hbWU6IHN0cmluZ1xuICBhbW91bnQxOiBzdHJpbmdcbiAgYW1vdW50Mjogc3RyaW5nXG4gIGFtb3VudDM6IG51bWJlclxufVxuXG5pbnRlcmZhY2UgU3VtbWFyeU1ldGhvZFByb3BzPFQgPSBQcm9kdWN0PiB7XG4gIGNvbHVtbnM6IFRhYmxlQ29sdW1uQ3R4PFQ+W11cbiAgZGF0YTogVFtdXG59XG5cbmNvbnN0IGdldFN1bW1hcmllcyA9IChwYXJhbTogU3VtbWFyeU1ldGhvZFByb3BzKSA9PiB7XG4gIGNvbnN0IHsgY29sdW1ucywgZGF0YSB9ID0gcGFyYW1cbiAgY29uc3Qgc3Vtczogc3RyaW5nW10gPSBbXVxuICBjb2x1bW5zLmZvckVhY2goKGNvbHVtbiwgaW5kZXgpID0+IHtcbiAgICBpZiAoaW5kZXggPT09IDApIHtcbiAgICAgIHN1bXNbaW5kZXhdID0gJ1RvdGFsIENvc3QnXG4gICAgICByZXR1cm5cbiAgICB9XG4gICAgY29uc3QgdmFsdWVzID0gZGF0YS5tYXAoKGl0ZW0pID0+IE51bWJlcihpdGVtW2NvbHVtbi5wcm9wZXJ0eV0pKVxuICAgIGlmICghdmFsdWVzLmV2ZXJ5KCh2YWx1ZSkgPT4gTnVtYmVyLmlzTmFOKHZhbHVlKSkpIHtcbiAgICAgIHN1bXNbaW5kZXhdID0gYCQgJHt2YWx1ZXMucmVkdWNlKChwcmV2LCBjdXJyKSA9PiB7XG4gICAgICAgIGNvbnN0IHZhbHVlID0gTnVtYmVyKGN1cnIpXG4gICAgICAgIGlmICghTnVtYmVyLmlzTmFOKHZhbHVlKSkge1xuICAgICAgICAgIHJldHVybiBwcmV2ICsgY3VyclxuICAgICAgICB9IGVsc2Uge1xuICAgICAgICAgIHJldHVybiBwcmV2XG4gICAgICAgIH1cbiAgICAgIH0sIDApfWBcbiAgICB9IGVsc2Uge1xuICAgICAgc3Vtc1tpbmRleF0gPSAnTi9BJ1xuICAgIH1cbiAgfSlcblxuICByZXR1cm4gc3Vtc1xufVxuXG5jb25zdCB0YWJsZURhdGE6IFByb2R1Y3RbXSA9IFtcbiAge1xuICAgIGlkOiAnMTI5ODcxMjInLFxuICAgIG5hbWU6ICdUb20nLFxuICAgIGFtb3VudDE6ICcyMzQnLFxuICAgIGFtb3VudDI6ICczLjInLFxuICAgIGFtb3VudDM6IDEwLFxuICB9LFxuICB7XG4gICAgaWQ6ICcxMjk4NzEyMycsXG4gICAgbmFtZTogJ1RvbScsXG4gICAgYW1vdW50MTogJzE2NScsXG4gICAgYW1vdW50MjogJzQuNDMnLFxuICAgIGFtb3VudDM6IDEyLFxuICB9LFxuICB7XG4gICAgaWQ6ICcxMjk4NzEyNCcsXG4gICAgbmFtZTogJ1RvbScsXG4gICAgYW1vdW50MTogJzMyNCcsXG4gICAgYW1vdW50MjogJzEuOScsXG4gICAgYW1vdW50MzogOSxcbiAgfSxcbiAge1xuICAgIGlkOiAnMTI5ODcxMjUnLFxuICAgIG5hbWU6ICdUb20nLFxuICAgIGFtb3VudDE6ICc2MjEnLFxuICAgIGFtb3VudDI6ICcyLjInLFxuICAgIGFtb3VudDM6IDE3LFxuICB9LFxuICB7XG4gICAgaWQ6ICcxMjk4NzEyNicsXG4gICAgbmFtZTogJ1RvbScsXG4gICAgYW1vdW50MTogJzUzOScsXG4gICAgYW1vdW50MjogJzQuMScsXG4gICAgYW1vdW50MzogMTUsXG4gIH0sXG5dXG48L3NjcmlwdD5cbiIsIlBsYXlncm91bmRNYWluLnZ1ZSI6IjxzY3JpcHQgc2V0dXA+XG5pbXBvcnQgQXBwIGZyb20gJy4vQXBwLnZ1ZSdcbmltcG9ydCB7IHNldHVwRWxlbWVudFBsdXMgfSBmcm9tICcuL2VsZW1lbnQtcGx1cy5qcydcbnNldHVwRWxlbWVudFBsdXMoKVxuPC9zY3JpcHQ+XG5cbjx0ZW1wbGF0ZT5cbiAgPEFwcCAvPlxuPC90ZW1wbGF0ZT5cbiIsImltcG9ydF9tYXAuanNvbiI6IntcbiAgXCJpbXBvcnRzXCI6IHtcbiAgICBcImVsZW1lbnQtcGx1c1wiOiBcImh0dHBzOi8vcHJldmlldy05OTc3LWVsZW1lbnQtcGx1cy5zdXJnZS5zaC9idW5kbGUvaW5kZXguZnVsbC5taW4ubWpzXCIsXG4gICAgXCJlbGVtZW50LXBsdXMvXCI6IFwidW5zdXBwb3J0ZWRcIlxuICB9XG59IiwiaW1wb3J0LW1hcC5qc29uIjoie1xuICBcImltcG9ydHNcIjoge1xuICAgIFwidnVlXCI6IFwiaHR0cHM6Ly91bnBrZy5jb20vQHZ1ZS9ydW50aW1lLWRvbUBsYXRlc3QvZGlzdC9ydW50aW1lLWRvbS5lc20tYnJvd3Nlci5qc1wiLFxuICAgIFwiQHZ1ZS9zaGFyZWRcIjogXCJodHRwczovL3VucGtnLmNvbS9AdnVlL3NoYXJlZEBsYXRlc3QvZGlzdC9zaGFyZWQuZXNtLWJ1bmRsZXIuanNcIixcbiAgICBcImVsZW1lbnQtcGx1c1wiOiBcImh0dHBzOi8vcHJldmlldy05OTc3LWVsZW1lbnQtcGx1cy5zdXJnZS5zaC9idW5kbGUvaW5kZXguZnVsbC5taW4ubWpzXCIsXG4gICAgXCJlbGVtZW50LXBsdXMvXCI6IFwidW5zdXBwb3J0ZWRcIixcbiAgICBcIkBlbGVtZW50LXBsdXMvaWNvbnMtdnVlXCI6IFwiaHR0cHM6Ly91bnBrZy5jb20vQGVsZW1lbnQtcGx1cy9pY29ucy12dWVAMi9kaXN0L2luZGV4Lm1pbi5qc1wiXG4gIH0sXG4gIFwic2NvcGVzXCI6IHt9XG59IiwiZWxlbWVudC1wbHVzLmpzIjoiaW1wb3J0IHsgZ2V0Q3VycmVudEluc3RhbmNlIH0gZnJvbSAndnVlJ1xuaW1wb3J0IEVsZW1lbnRQbHVzIGZyb20gJ2VsZW1lbnQtcGx1cydcblxubGV0IGluc3RhbGxlZCA9IGZhbHNlXG5hd2FpdCBsb2FkU3R5bGUoKVxuXG5leHBvcnQgZnVuY3Rpb24gc2V0dXBFbGVtZW50UGx1cygpIHtcbiAgaWYgKGluc3RhbGxlZCkgcmV0dXJuXG4gIGNvbnN0IGluc3RhbmNlID0gZ2V0Q3VycmVudEluc3RhbmNlKClcbiAgaW5zdGFuY2UuYXBwQ29udGV4dC5hcHAudXNlKEVsZW1lbnRQbHVzKVxuICBpbnN0YWxsZWQgPSB0cnVlXG59XG5cbmV4cG9ydCBmdW5jdGlvbiBsb2FkU3R5bGUoKSB7XG4gIHJldHVybiBuZXcgUHJvbWlzZSgocmVzb2x2ZSwgcmVqZWN0KSA9PiB7XG4gICAgY29uc3QgbGluayA9IGRvY3VtZW50LmNyZWF0ZUVsZW1lbnQoJ2xpbmsnKVxuICAgIGxpbmsucmVsID0gJ3N0eWxlc2hlZXQnXG4gICAgbGluay5ocmVmID0gJ2h0dHBzOi8vcHJldmlldy05OTc3LWVsZW1lbnQtcGx1cy5zdXJnZS5zaC9idW5kbGUvaW5kZXguY3NzJ1xuICAgIGxpbmsuYWRkRXZlbnRMaXN0ZW5lcignbG9hZCcsIHJlc29sdmUpXG4gICAgbGluay5hZGRFdmVudExpc3RlbmVyKCdlcnJvcicsIHJlamVjdClcbiAgICBkb2N1bWVudC5ib2R5LmFwcGVuZChsaW5rKVxuICB9KVxufSIsIl9vIjp7InNob3dIaWRkZW4iOnRydWUsInN0eWxlU291cmNlIjoiaHR0cHM6Ly9wcmV2aWV3LTk5NzctZWxlbWVudC1wbHVzLnN1cmdlLnNoL2J1bmRsZS9pbmRleC5jc3MifX0=)

Maybe you can refer to the `table-header` modification

**Review by tuongtd — COMMENTED** [review]
Great!

**Review by tuojxjnk — COMMENTED** [review]
OK

**Comment by tuongtd** [comment]
Why no one merge to fix this issue? :s

**Comment by piscesvivian** [comment]
Is there some merge this commit?

**Comment by tolking** [comment]
Fixed by #14315

**Pull request closed without merging**